src/Kreta/TaskManager/Domain/Model/Organization/Organization.php

```diff
@@ -46,4 +55,34 @@ public function slug() : OrganizationSlug
     {
         return $this->slug;
     }
+
+    public function owners() : array
+    {
+        return $this->owners->toArray();
+    }
+
+    public function addOwner(Owner $owner)
+    {
+        $this->owners->add($owner);
+    }
+
+    public function removeOwner(Owner $owner)
+    {
+        $this->owners->remove($owner);
+    }
+
+    public function participants() : array
+    {
+        return $this->participants->toArray();
+    }
+
+    public function addParticipant(Participant $participant)
```

Comment:
We should rename this addParticipant to improve the language ubiquity. What about "assignParticipant" (this is the term that Vaughn Vernon uses :P)